spec/controllers/api/v1/companies_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::CompaniesController, type: :controller do
+
+  describe "GET #show" do
+    let(:response_body) { JSON.parse(response.body) }
+    let(:company) { create(:company) }
+
```

Comment:
@Claudia108 @JaredRoth that's just how you can assign variables instead of using `before` in RSpec

http://stackoverflow.com/questions/5974360/rspec-difference-between-let-and-before-block